app/controllers/systems_controller.rb

```diff
@@ -12,6 +12,13 @@ def edit
   def update
     @system = System.find(params[:id])
     if @system.update(system_params)
+      @system.units.each do |unit|
+        unless current_user == @system.gm ||
+          current_user == unit.faction.user &&
+          @system.links.where(destination_system: target).count > 0
```

Comment:
Use 4 (not 0) spaces for indenting a condition in an `unless` statement spanning multiple lines.